tests/test_catalogs.py

```diff
@@ -0,0 +1,158 @@
+import requests
+from rdflib import Graph
+from rdflib.compare import isomorphic
+import json
+
+from os import environ as env
+HOST_URL = env.get("HOST_URL") + ":" + env.get("HOST_PORT")
+
+def test_catalogs_with_json():
+    "GET request to url returns a 200"
+    url = HOST_URL + "/catalogs"
```

Comment:
Også mulig å formattere på den måten

```suggestion
    url = f"{HOST_URL}/catalogs"
```

Reply:
Nice! Fiksa.